redspot.config.ts

```diff
@@ -7,7 +7,7 @@ export default {
     defaultNetwork: 'development',
     contract: {
         ink: {
-            toolchain: 'nightly',
+            toolchain: 'nightly-2022-07-24',
```

Comment:
If you lock toolchain here can you also add an rust-toolchain.toml ?
https://github.com/AstarNetwork/Astar/blob/master/rust-toolchain.toml

Reply:
Fixed as of https://github.com/AstarNetwork/astar-wasm-contracts-examples/pull/7/commits/cec6767869c39e6d6664f176b9a6bc5295d6af9a